### PR TITLE
refactoring Modal to Ng2Modal ng2-modal

### PR DESCRIPTION
**Pull request by gravity-addiction**

### CODE REVIEW

File: src/ng2-modal.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {Component, Input, Output, EventEmitter, ElementRef, ViewChild} from "@angular/core";
 
 @Component({
-    selector: "modal",
+    selector: "ng2-modal",
```
Comment (pleerock) [inline src/ng2-modal.ts]: I prefer file to be named `ng2-modal`, but why do you think `<ng2-modal>` component name is a better name then `<modal>`? `<modal>` is simple and straightforward.

Comment (gravity-addiction) [inline src/ng2-modal.ts]: I don't think it's any better. It's compliant for the upcoming webcomponents spec draft from w3c. In the new draft any custom elements must contain a dash. The reason they claim is so there will be less conflicts between spec elements and user custom elements.

Comment (pleerock) [inline src/ng2-modal.ts]: I dont see any good reason to rename it right now. I would wait for upcoming webcomponents spec draft and see if others would follow it too. 

> The reason they claim is so there will be less conflicts between spec elements and user custom elements.
I think this is more recommendation rather then a practical reason. In practice updates come from those guys rarely once per few years and more. In such period we will see angular 3 and new version of modal maybe with different name if it conflicts

File: src/ng2-modal.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import {Component, Input, Output, EventEmitter, ElementRef, ViewChild} from "@angular/core";
```
Comment (pleerock) [inline src/ng2-modal.ts]: in latest push I renamed ng2-modal to `index.ts`, and right now users will import like this: `import {Modal} from "ng2-modal";`

**Comment by pleerock** [comment]
I decided to left it as now. Thank you for your effort.

**Pull request closed without merging**